portfolio/src/main/webapp/scripts/templatemo-script.js

```diff
@@ -46,37 +46,39 @@ $(document).ready(function () {
     document.querySelector('.tm-current-year').textContent = new Date().getFullYear();
 });
 
-var pictures = new Array("images/apollo.jpg", "images/bob.jpeg", "images/boomer.jpg", "images/drift.jpg",
-                        "images/filbert.jpg", "images/kiki.jpg", "images/lolly.jpg", "images/punchy.jpg",
-                        "images/rosie.jpg");
+var pictures = new Array(new Array("images/blathers-negative.png", "images/isabelle-negative.png"), 
+                        new Array("images/blathers-neutral.png", "images/celeste-neutral.png", "images/isabelle-neutral.png"), 
+                        new Array("images/blathers-positive.png", "images/celeste-positive.png", "images/isabelle-positive.png"));
 
 /**
  * Fetch the comment data in Json format and create a comment box in the comments section for each comment.
  */
 async function fetchJson() {
     const response = await fetch('/data');
-    const json = await response.json();
+    const jsonData = await response.json();
 
     var template = document.querySelector('#comment-template');
     var commentBox = template.content.querySelector('div');
     var temp;
 
-    json.forEach(comment => {
-        temp = modifyTemplate(commentBox, comment);
-        document.querySelector('#comments').appendChild(temp);
+    Object.keys(jsonData).forEach(function(key) {
```

Comment:
you might need to clear `#comment-list` beforehand, in case this function is called multiple times.

Reply:
I've made the change